### PR TITLE
Changed The Background color in choose your platform section

### LINKED ISSUE #1735: Add a better background color to Choose your platform section on homepage
### Current Behavior
The background color of a section that comes up when you click on any of the platforms under the `Choose your platform` section is the same as that of the whole section. 
image

### Desired Behavior
Change the background color of that inner section that pops up

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by shailpujan88**
**Description**

This PR fixes #1735

**Notes for Reviewers**
Changed the background color of the Choose Your Platform section.

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by shailpujan88** [comment]
image
@leecalcote I have made the required changes. Can you please verify them?

File: Gemfile.lock
Comment (ritiksaxena124) [inline Gemfile.lock:1]: Don't push `Gemfile.lock` file. Revert this change

File: _includes/getting-started.html [hunk 1]
```diff
@@ -124,7 +124,7 @@ <h3>Brew User:</h3>
       </div>
 
       <!-- Docker User -->
-      <div id="docker_code" class="toggle hideDiv">
+      <div id="docker_code" class="toggle hideDiv" style="background-color: rgb(84, 89, 91);">
```
Comment (ritiksaxena124) [inline _includes/getting-started.html:127]: Don't use hardcoded values for styling. Try to use variables and avoid using inline-styling.

File: _includes/getting-started.html [hunk 2]
```diff
@@ -109,7 +109,7 @@ <h3>Bash using Mac or Linux</h3>
       </div>
 
       <!-- Brew -->
-      <div id="brew_code" class="toggle hideDiv">
+      <div id="brew_code" class="toggle hideDiv" style="background-color: rgb(84, 89, 91);">
```
Comment (ritiksaxena124) [inline _includes/getting-started.html:112]: Here as well.

File: _includes/getting-started.html [hunk 3]
```diff
@@ -149,7 +149,7 @@ <h4 style="font-size: 1.5em;"> Use Docker Extension for Meshery</h4>
         </div>
       </div>
       <!-- Kubernetes -->
-      <div id="k8s-instructions_code" class="toggle hideDiv">
+      <div id="k8s-instructions_code" class="toggle hideDiv" style="background-color: rgb(84, 89, 91);">
```
Comment (ritiksaxena124) [inline _includes/getting-started.html:152]: Same

**Review by ritiksaxena124 — CHANGES_REQUESTED** [review]

**Comment by ritiksaxena124** [comment]
Don't use hardcoded values for styling. Try to use variables and avoid using inline styling @shailpujan88.

**Pull request closed without merging**